cli/cmd/utils.go

```diff
@@ -188,3 +196,38 @@ func askForConfirmation(message string) bool {
 	fmt.Scanln(&input)
 	return input == "yes"
 }
+
+func waitForDeployment(cs *kubernetes.Clientset, ns, name string, retryInterval, timeout time.Duration) {
```

Comment:
Is there a better place for storing this code than `utils.go`? Packages and source files named `utils` tend to become a messy "catch-all" place for storing anything we don't know where to store. In my experience if you feel like you have no better place for the code, it hints at a wrong abstraction. Fixing a wrong abstraction is very likely outside the scope of this PR, but I'd still suggest we think about minimizing our usage of stuff called "utils" rather than adding to it.